src/store/reducer.js

```diff
@@ -8,7 +8,7 @@ const {
 
 const initialState = {
   assetRates: {}, // assetId -> rate
-  assets: { ETH: 0, DAI: 1, NXM: 255 },
+  assets: { ETH: 0, DAI: 1, USDC: 6, NXM: 255 },
```

Comment:
also is there a reason why we keyed  the assets off the assetSymbol? cover-router usually receives assetId. this line in selectAssetSymbol function could be made simpler if its keyed off assetId:

```
  return Object.keys(assets).find(key => assets[key] === assetId); // keyed off assetSymbol
  return assets[assetId]; // keyed off assetId
```

Reply:
We use it for readability